src/app/devices/addDevice/state.ts

```diff
@@ -1,3 +1,7 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
```

Comment:
do we have to add this on every file?

Reply:
According to the policy, yes. Maybe something like this is worth looking into: https://stackoverflow.com/questions/38823319/automatically-add-copyright-banner-in-vs-code